datasets/caldor-fire-characteristics-burn-severity.data.mdx

```diff
@@ -0,0 +1,69 @@
+---
+id: caldor-fire-behavior-burn-severity
+name: "Caldor Fire Behavior and Burn Severity"
+description: "Assets describing the progression and active fire behavior of the 2021 Caldor Fire in California."
+media:
+  src: ::file ./caldor-fire--dataset-cover.jpg
+  alt: Hillside engulfed by a wildfire.
+  author:
+    name: Marek Piwnicki
+    url: https://unsplash.com/photos/WiZOyYqzUss
+thematics:
+  - eis
+layers:
+  - id: caldor-fire-behavior
+    stacCol: caldor-fire-behavior
+    name: Fire Behavior
+    type: raster
+    description: ""
+    initialDatetime: newest
+    zoomExtent:
+      - 10
+      - 20
+    sourceParams:
+      colormap_name: inferno_r
+      rescale:
+        - -41
+        - 98
```

Comment:
```suggestion
        - 0
        - 93
```

Reply:
@slesaad thank you for making all these changes! I have one other note related to the data scaling: the burn severity raster only has valid values from 1-4. Is it possible to change those values accordingly? I also saw your note on whether `inferno` or `inferno_r` would be better... honestly I think either work, but your comment about warmer colors indicating higher severity values makes sense to me. 

Last thing: is it possible to replace "Fire Behavior" with "Mean Fire Radiative Power (MW)"?